codalab/worker/file_util.py

```diff
@@ -11,7 +11,20 @@
 from codalab.common import BINARY_PLACEHOLDER
 
 NONE_PLACEHOLDER = '<none>'
-GIT_PATTERN = '.git'
+
+# Patterns to always ignore when zipping up directories
+ALWAYS_IGNORE_PATTERNS = ['.git', '._*']
```

Comment:
Do we need to do something to ignore `__MACOSX`?

Reply:
I actually don't get any `__MACOSX` folders when zipping, but I will add it to this list and write a test for it too.

Reply:
For some reason, I can reproduce getting `._` files, but not the `__MACOSX` folder. I read some posts online and it seems like calling `bsdtar` with `--exclude __MACOSX`  should work.